api/v1alpha1/cluster_types.go

```diff
@@ -33,6 +33,7 @@ type CrdbClusterSpec struct {
 	// (Required) Number of nodes (pods) in the cluster
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Number of nodes",xDescriptors="urn:alm:descriptor:com.tectonic.ui:podCount"
 	// +required
+	// +kubebuilder:validation:Minimum=3
```

Comment:
hmmm, I think we still want to support making single node clusters?

We just can't support 2 nodes or go from 3 nodes to 1.